chall06/phakakos.py

```diff
@@ -0,0 +1,114 @@
+#!/usr/bin/env python3
+
+import sys
+import fileinput
+
+def errnoHandle(code, source):
+        #replace with (sys.argv[0] + " ") for flexible use. My enviroment does not match the tests, which gives a different out put, so I'm just gonna hardcode it :^)
+        beginning = "./phakakos.py "
+        if code == -1:
+                print(beginning + source + ": Can't read file")
+        elif code == -2:
+                print(beginning + source + ": Not enough space in the given shelves")
```

Comment:
nice!

how comes your environment does not match the tests? What was the output of your environment? and what is your environment?

Reply:
I used Windows and run it thru the Python Shell so it gives me the full path to the file rather than the "./name.py". It should give the correct output in Unix, but left it hardcoded for now.